src/modpacks/models.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import unicode_literals
+from django.db import models
+
+
+class Mod(models.Model):
+    """
+    A mod that can be combined to make a modpack
+    """
+    mod_name = models.CharField(verbose_name='Mod name',
+                                max_length=200)
+    version = models.CharField(verbose_name='Mod version',
+                               blank=True,
+                               max_length=10)
+    author = models.CharField(verbose_name='Mod author',
+                              blank=True,
+                              max_length=200)
+    url = models.URLField(verbose_name='Link to mod website',
+                          blank=True)
+
+    def __str__(self):
+        return self.mod_name
+
+
+class Modpack(models.Model):
```

Comment:
Multiple classes in a file is generally frowned upon. Consider a models package (folder with \_\_init\_\_.py)